proposals/bound-callable-references.md

```diff
@@ -128,10 +228,31 @@ We could try resolve type in LHS first, and only then try expression. This has t
 
 ## Possible future advancements
 
```

Comment:
No text between two headers is generally a bad sign :)

I'd explain briefly here what the meaning of this section is: we are aware of these possibilities, but probably won't implement them in 1.1